salt/states/service.py

```diff
@@ -418,21 +423,24 @@ def mod_watch(name, sfun=None, sig=None, reload=False, full_restart=False):
             return ret
     elif sfun == 'running':
         if __salt__['service.status'](name, sig):
+            past_participle = None
```

Comment:
is that me or there is a race here, past_particiciple should at least be defined 2 lines upper.